studio/components/layouts/AuthLayout/AuthLayout.utils.ts

```diff
@@ -4,37 +4,26 @@ import { ProductMenuGroup } from 'components/ui/ProductMenu/ProductMenu.types'
 export const generateAuthMenu = (ref: string): ProductMenuGroup[] => {
   return [
     {
-      title: 'General',
-      items: [
-        { name: 'Users', key: 'users', url: `/project/${ref}/auth/users`, items: [] },
-        { name: 'Policies', key: 'policies', url: `/project/${ref}/auth/policies`, items: [] },
-        ...(IS_PLATFORM
-          ? [
-              {
-                name: 'Logs',
-                key: 'logs',
-                url: `/project/${ref}/auth/logs`,
-                items: [],
-              },
-            ]
-          : []),
-      ],
+      title: 'Users',
```

Comment:
this look right? @MildTomato 
<img width="287" alt="image" src="https://user-images.githubusercontent.com/19742402/196589766-281983bc-a251-48cc-b03d-23807be5809b.png">